data/forms/medical-needs.json

```diff
@@ -1,5 +1,5 @@
 {
-  "heading": "Medical Needs",
+  "heading": "Do you have any significant medical needs that affect your housing requirements?",
```

Comment:
In the designs, there is a variation for this wording - do you, does <x>. I think it's ok like this however.

Reply:
Is there a way for the JSON data to be contextual to the point in the data capture process?